<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Surface_RenderTopo_TabWidget</class>
 <widget class="QWidget" name="Surface_RenderTopo_TabWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>337</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="0" colspan="3">
    <widget class="QCheckBox" name="check_drawPhi1">
     <property name="text">
      <string>draw phi1</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="4">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="3">
    <widget class="ColorComboBox" name="combo_phi1Color"/>
   </item>
   <item row="3" column="3">
    <widget class="ColorComboBox" name="combo_phi2Color"/>
   </item>
   <item row="1" column="3">
    <widget class="ColorComboBox" name="combo_dartsColor"/>
   </item>
   <item row="3" column="0" colspan="3">
    <widget class="QCheckBox" name="check_drawPhi2">
     <property name="text">
      <string>draw phi2</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="13" column="0" colspan="2">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>explod faces</string>
     </property>
    </widget>
   </item>
   <item row="13" column="2" colspan="2">
    <widget class="QSlider" name="slider_facesScaleFactor">
     <property name="value">
      <number>50</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="10" column="2" colspan="2">
    <widget class="QComboBox" name="combo_positionAttribute">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <item>
      <property name="text">
       <string>- select attribute -</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="12" column="2" colspan="2">
    <widget class="QSlider" name="slider_edgesScaleFactor">
     <property name="value">
      <number>50</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="15" column="0" colspan="4">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>156</width>
       <height>161</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="6" column="0" colspan="3">
    <widget class="Line" name="line_2">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>230</width>
       <height>0</height>
      </size>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="3">
    <widget class="QCheckBox" name="check_drawDarts">
     <property name="text">
      <string>draw darts</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="12" column="0" colspan="2">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>explod edges</string>
     </property>
    </widget>
   </item>
   <item row="10" column="0" colspan="2">
    <widget class="QLabel" name="label_2">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Position :</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0" colspan="4">
    <widget class="QLabel" name="label_map_param">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Per map parameters</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ColorComboBox</class>
   <extends>QComboBox</extends>
   <header>colorComboBox.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>check_drawDarts</tabstop>
  <tabstop>combo_dartsColor</tabstop>
  <tabstop>check_drawPhi1</tabstop>
  <tabstop>combo_phi1Color</tabstop>
  <tabstop>check_drawPhi2</tabstop>
  <tabstop>combo_phi2Color</tabstop>
  <tabstop>slider_edgesScaleFactor</tabstop>
  <tabstop>slider_facesScaleFactor</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
